libraries/jevmodal/media/js/jevmodal.js

```diff
@@ -125,7 +127,7 @@ function addJevModalHtml (id){
 			+'</div>';
 		}
 		else {
-			myModal = '<div class="modal  hide fade ' + modalsize + ' jevmodal" id="'+id+'" tabindex="-1" role="dialog" aria-labelledby="'+id+'Label" aria-hidden="true" >'
```

Comment:
@carcam We use 'fade' class in  other places event repeat save popup dialog, event edit modal etc.  Why is this situation different?

Reply:
To be honest, this is a change from 2019. I was able to reproduce the issue in my development box so I see it like a generic problem. If you want I can reproduce the issue again and do some research again.

But the main problem was that JS was not triggering. I'm a bit surprised the user didn't come back with all latest releases...